### PR TITLE
[bitnami/magento] Update chown of the deployment

### LINKED ISSUE #17337: [bitnami/magento] volume permissions is not working
### Name and Version

bitnami/magento 22.1.2

### What architecture are you using?

amd64

### What steps will reproduce the bug?

Install magento with  `volumePermissions.enabled=true`

### Are you using any custom parameters or values?

```yaml
magento:
  volumePermissions:
    enabled: true
```

### What is the expected behavior?

Volume permissions should be successfully updated

### What do you see instead?

`volume-permissions` init container is failing with this error in logs: 
```
chown: missing operand after '/bitnami/magento'                                                                                                                                                                     
Try 'chown --help' for more information. 
```

### Additional information

I think it is failing because `chown` is missing reference to a user and group
```
          args:
            - -ec
            - |
              mkdir -p "/bitnami/magento"
              chown "/bitnami/magento"
              find "/bitnami/magento" -mindepth 1 -maxdepth 1 -not -name "config" -not -name ".snapshot" -not -name "lost+found" | xargs chown -R {{ .Values.containerSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }}
          securityContext:
```

### PR DESCRIPTION
**Pull request by Mauraza**
### Description of the change

Update chown with the user:group 

### Applicable issues

- fixes #17337

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/magento/templates/deployment.yaml [hunk 1]
```diff
@@ -83,7 +83,7 @@ spec:
             - -ec
             - |
               mkdir -p "/bitnami/magento"
-              chown "/bitnami/magento"
+              chown -R {{ .Values.containerSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }} /bitnami/magento
```
Comment (joancafom) [inline bitnami/magento/templates/deployment.yaml:86]: Why are you performing the operation recursively? The command below seems to do the same thing, but avoiding some other paths...

File: .vib/magento/cypress/cypress/support/commands.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ Cypress.Commands.add('logout', () => {
 
 Cypress.on('uncaught:exception', (err, runnable) => {
   // we expect an application error with message 'rendering locks'
-  // and don't want to fail the test so we return false
+  // and don't want to fail the test so we return false XMLHttpRequest
```
Comment (joancafom) [inline .vib/magento/cypress/cypress/support/commands.js:48]:
```suggestion
  // and don't want to fail the test so we return false
```

File: bitnami/magento/templates/deployment.yaml (hunk 1 above)
Comment (Mauraza) [reply bitnami/magento/templates/deployment.yaml]: I updated that line removing the '-R'

**Review by joancafom — APPROVED** [review]
LGTM!

**Pull request merged**